docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -206,8 +206,14 @@ If you need to use an integration that you have previously revoked, log in again
 
 ## Why does Codacy ask for permission to create SSH keys?
 
-When you add a private repository to Codacy, Codacy uses the integration with your Git provider to create a new SSH key on the repository. Codacy then uses that SSH key every time it needs to clone the repository.
+!!! info "This section applies only to GitLab and Bitbucket"
+    For GitHub, Codacy creates an SSH key on the repository **exclusively as a fallback mechanism** when the Contents permission is missing. To avoid disruptions in the future, make sure you accept Codacy GitHub App [updated permissions](#github-cloud) on your GitHub organization.
```

Comment:
Does this admonition break our usual structure? I remember using custom titles only with empty `info` admonitions.

Either way, I'm OK with using it like this here. This is temporary I presume?

Reply:
You are right, this is the first time we add additional content to an `info` admonition. But at the same time, we always use `info` admonitions to say "This page/section only applies to <something>"...

So I opted to break our usual standard by adding more information (because it's temporary), instead of breaking by using a different admonition type. 🤓 

I would keep it like this, as we'll be removing the additional temporary content in short/medium future.

WDYT?

Reply:
Sounds good, especially if this is temporary 👍